enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/api/WebSocketAPI.java

```diff
@@ -256,6 +266,8 @@ public WebSocketThrottleResponse processFramedata(RequestContext requestContext)
         }
         WebSocketThrottleResponse webSocketThrottleResponse = new WebSocketThrottleResponse();
         webSocketThrottleResponse.setOverLimitState();
+        webSocketThrottleResponse.setApimErrorCode(Integer.parseInt(requestContext.getProperties()
+                .get(APIConstants.MessageFormat.ERROR_CODE).toString()));
```

Comment:
```
requestContext.getProperties() .get(APIConstants.MessageFormat.ERROR_CODE)
```
can this be null? if so avoid NPE

Reply:
This can't be null. It is always an empty map when instantiated.

Reply:
i mean, `requestContext.getProperties().get(APIConstants.MessageFormat.ERROR_CODE)` can this be null. we are calling .toString() on a null object can throw a NPE